src/OSPSuite.Core/Domain/Builder/IndividualParameter.cs

```diff
@@ -1,25 +1,6 @@
-﻿using OSPSuite.Core.Domain.Services;
-
-namespace OSPSuite.Core.Domain.Builder
+﻿namespace OSPSuite.Core.Domain.Builder
 {
-   public class IndividualParameter : PathAndValueEntity, IWithDefaultState
+   public class IndividualParameter : ParameterValue
```

Comment:
I have changed the hiearchie a bit. IndividualParameter and ExpressionParameter are now ParameterValue